persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/PfamDomainRepositoryImpl.java

```diff
@@ -1,247 +0,0 @@
-package org.cbioportal.genome_nexus.persistence.internal;
-
-import com.univocity.parsers.common.processor.BeanListProcessor;
-import com.univocity.parsers.csv.CsvParser;
-import com.univocity.parsers.csv.CsvParserSettings;
-import org.cbioportal.genome_nexus.model.PfamA;
-import org.cbioportal.genome_nexus.model.PfamDomain;
-import org.cbioportal.genome_nexus.persistence.PfamDomainRepository;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.core.io.ClassPathResource;
-import org.springframework.core.io.Resource;
-import org.springframework.stereotype.Component;
-
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.Reader;
-import java.io.UnsupportedEncodingException;
-import java.util.*;
-
-/**
- * @author Selcuk Onur Sumer
- */
-@Component("defaultPfamDomainRepository")
-public class PfamDomainRepositoryImpl implements PfamDomainRepository
```

Comment:
We don't need a custom repository implementation anymore, we use `MongoRepository`